timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -50,38 +51,102 @@ export class TimeGraphChart extends TimeGraphChartLayer {
     protected afterAddToContainer() {
         this.shiftKeyDown = false;
         this.ctrlKeyDown = false;
+
```

Comment:
Add a method: setControls(map<'string', list<keycode>>)
with the values 
'zoomin'
'zoomout'
'panleft'
'panright'

then....